mender-update/daemon/states.cpp

```diff
@@ -48,6 +48,7 @@ class DefaultStateHandler {
 
 static void DefaultAsyncErrorHandler(sm::EventPoster<StateEvent> &poster, const error::Error &err) {
 	if (err != error::NoError) {
+		log::Error(err.String());
```

Comment:
Q: Do the states add enough context here, or do you think we should add a note on what's up here?

Reply:
All of the Update Module calls already include the state as context in the error, so I think this is more or less covered.